tools/pom.xml

```diff
@@ -54,14 +54,17 @@
         <dependency>
             <groupId>org.apache.cassandra</groupId>
             <artifactId>cassandra-all</artifactId>
+            <version>3.11.10</version>
```

Comment:
Why do you overwrite the dependency version that is managed in the parent pom.xml?

Reply:
Fixed in 55f5602bdf2b9c2479ffcb8114708b5166684dab commit